### ISSUE TITLE
Can't Run Android Emulator on 1.5.2 thru 1.6.1: Multiple dex files define LayoutBase

### ISSUE THREAD

**Issue by vbresults** [NativeScript/NativeScript#1596]
```
$ tns emulate android
```

```
:preDexDebug UP-TO-DATE
:dexDebug
Unknown source file : UNEXPECTED TOP-LEVEL EXCEPTION:
Unknown source file : com.android.dex.DexException: Multiple dex files define Lorg/nativescript/widgets/LayoutBase;
Unknown source file : 	at com.android.dx.merge.DexMerger.readSortableTypes(DexMerger.java:579)
Unknown source file : 	at com.android.dx.merge.DexMerger.getSortedTypes(DexMerger.java:535)
Unknown source file : 	at com.android.dx.merge.DexMerger.mergeClassDefs(DexMerger.java:517)
Unknown source file : 	at com.android.dx.merge.DexMerger.mergeDexes(DexMerger.java:164)
Unknown source file : 	at com.android.dx.merge.DexMerger.merge(DexMerger.java:188)
Unknown source file : 	at com.android.dx.command.dexer.Main.mergeLibraryDexBuffers(Main.java:504)
Unknown source file : 	at com.android.dx.command.dexer.Main.runMonoDex(Main.java:334)
Unknown source file : 	at com.android.dx.command.dexer.Main.run(Main.java:277)
Unknown source file : 	at com.android.dx.command.dexer.Main.main(Main.java:245)
Unknown source file : 	at com.android.dx.command.Main.main(Main.java:106)

:dexDebug FAILED

FAILURE: Build failed with an exception.
```

**Comment by vjoao** [comment]
Just remove and add the platform. This sometimes happens to me also.

**Comment by vbresults** [comment]
@vjoao Thanks, that fixed it

**Comment by vbresults** [comment]
This issue is back, and adding/removing the platform does not fix it :/

**Comment by Jayuda** [comment]
i have same error ? I have remove and add the platform. But This not solved it

**Comment by vbresults** [comment]
@Jayuda The way I fixed it was by installing 1.1.61, deleting the `platforms` folder, `node_modules` folder, deleting `package.json` and then creating a new `package.json` with `tns init --force`, then re-add your platforms.

**Comment by Jayuda** [comment]
@vbresults : Thanks a lot. Its SOLVED. GBU
